cmd/cored/main.go

```diff
@@ -196,9 +207,9 @@ func runServer() {
 	if err != nil {
 		chainlog.Fatalkv(ctx, chainlog.KeyError, err)
 	}
-	resetInDevIfRequested(db)
+	resetInDevIfRequested(db, raftDB)
 
-	conf, err := config.Load(ctx, db)
+	conf, err := config.Load(ctx, db, raftDB)
```

Comment:
Oof, yeah, this is a good point. I've been trying to punt on "polishing" work but this will break systems with the config in postgres, so my inclination is to try to squeak that into this PR